keyboards/ergodox_ez/keymaps/skug/config.h

```diff
@@ -0,0 +1,12 @@
+#ifndef CONFIG_USER_H
```

Comment:
Could you replace the Boiler Plate with `#pragma once`? 

Eg, the entire file should look like: 
```c
#pragma once

#undef LSPO_KEY
#undef RSPC_KEY
#define LSPO_KEY KC_8 // Nordic Left SpaceCadet
#define RSPC_KEY KC_9 // Nordic Right SpaceCadet

```